_posts/2021-07-07-cockpit-248.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Cockpit 248
+author: garrett
+date: '2021-07-07'
+tags: cockpit, machines, ostree, podman, grafana, pcp, redis, metrics, pam
+slug: cockpit-248
+category: release
+summary: 'Metrics, PAM, Machines, OSTree, and git main branch changes'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 248, cockpit-machines 247, cockpit-ostree 187, and cockpit-podman 32:
+
+
+## Metrics: Install missing packages
+
+[Cockpit 247](https://cockpit-project.org/blog/cockpit-247.html) introduced setting up a machine for Grafana metrics. However, this only worked if the `pcp` and `redis` packages were already installed. In Cockpit 248 and after, selecting these options will install any missing packages.
+
+![screenshot of install missing packages]({{ site.baseurl }}/images/248-metrics-install-missing-packages.png)
+
+![screenshot of install missing packages]({{ site.baseurl }}/images/248-metrics-install-missing-packages-2.png)
+
+This completes the story of [setting up PCP and Grafana metrics with Cockpit](https://cockpit-project.org/blog/pcp-grafana.html).
+
+## PAM: Deprecate `pam_cockpit_cert` module
+
+The `pam_cockpit_cert` module is deprecated and no longer required for client certificate authentication. Cockpit continues to ship a stub module which returns `PAM_IGNORE`, but a future release will drop this stub.
+
+Upgrading packages will not update a modified `/etc/pam.d/cockpit` file. If you have modified this file, please take care to remove the relevant entry.
+
+## Machines: Bug fixes and improvements
+
+Cockpit-machines has a new release with several bug fixes and minor improvements.
```

Comment:
I'd write small "c" at the start of this sentence, but that's really a matter of personal taste.